build.gradle

```diff
@@ -68,6 +68,7 @@ dependencies {
 	implementation group: "org.hashids", name: "hashids", version: "1.0.3"
 	implementation group: "org.liquibase", name: "liquibase-core", version: "4.1.0"
 	implementation group: "org.springframework.cloud", name: "spring-cloud-gcp-starter-trace", version: "1.2.5.RELEASE"
+  implementation group: "org.springframework.cloud", name: "spring-cloud-gcp-starter-logging", version: "1.2.5.RELEASE"
```

Comment:
Nit: indentation, alphabetize

Reply:
> So, sorry, I'm confused, how does this mix with opencensus in #111? Or it's log specific, nothing to do with traces...?

This is log specific and shouldn't affect traces at all (though yeah, the two libraries have pretty similar names...). This is updating WM's log format to let us use things like Error Reporting [[AS-449]](https://broadworkbench.atlassian.net/browse/AS-449) and generally to have more structured logs